better_bookmarks/manifest.json

```diff
@@ -6,7 +6,8 @@
     "permissions": [
         "bookmarks",
         "contextMenus",
-        "activeTab"
+        "activeTab",
+		"storage"
```

Comment:
Why is storage indented more than everything at its logical level?  It needs to be at the same level as ```activeTab```, ```contextMenus```, and ```bookmarks```.  If we're editing each other's code, we need to make sure that we stick to 4 spaces of indentation, no tabs, and keep everything that's at the same logical level indented the same amount.

Reply:
It doesn't look like the indentation has been fixed here; move ```"storage"``` back so that it lines up with the rest of the strings in the ```permissions``` list.